build.ps1

```diff
@@ -292,8 +297,6 @@ try
         $testCategory += "TestCategory=Unit"
         $testCategory += "|"
         $testCategory += "TestCategory=E2E"
-        $testCategory += "|"
```

Comment:
Moved invalid cert tests to nightly E2E test-runs

Reply:
Because 5-minute overhead for docker operations and low likelihood of regression. Correct?

Reply:
Correct, it is extra time for setting up the environment currently. There are things that can be further streamline if we need to run it as a PR gate with less than a minute of setup time if all the extra pings and changes are removed.